Tests/AdyenTests/Adyen Tests/Components/Affirm/AffirmComponentTests.swift

```diff
@@ -58,7 +58,7 @@ class AffirmComponentTests: XCTestCase {
         sut.deliveryAddressToggleItem.value = false
         
         // When
-        let paymentDetails = sut.createPaymentDetails()
+        let paymentDetails = try sut.createPaymentDetails()
```

Comment:
Now that this function can throw, let's tests a few failed scenarios with missing data